lib/linguist/languages.yml

```diff
@@ -44,6 +44,16 @@ AGS Script:
   tm_scope: source.c++
   ace_mode: c_cpp
 
+Alex Haskell:
+  type: programming
+  group: Haskell
```

Comment:
If that's all it takes, great :) I just modeled it on C2hs Haskell since I saw that preprocessor was broken out in the `languages.yml`. I'd imagine that the heuristics for identifying Haskell would identify Alex Haskell about as well as C2hs Haskell (that is, often extremely well but perhaps occasionally things go off the rails)